src/services/logger.js

```diff
@@ -0,0 +1,30 @@
+const { createLogger, format, transports } = require('winston');
+
+const { combine, timestamp, prettyPrint } = format;
+
+/**
+ *  
+ *  levels: {
+    error: 0,
+    warn: 1,
+    info: 2
+    }
+ * 
+ * 
+ */
```

Comment:
Is this comment required?

Reply:
I added this for development phase actually. I can remove it.

Reply:
Yup I think it can be removed, since we are using only info and error logs.